{
    "config": {
        "abort": {
            "already_configured": "A fi\u00f3k m\u00e1r konfigur\u00e1lva van",
            "already_in_progress": "A konfigur\u00e1l\u00e1s m\u00e1r folyamatban van",
            "authorize_url_timeout": "Id\u0151t\u00fall\u00e9p\u00e9s a hiteles\u00edt\u00e9si URL l\u00e9trehoz\u00e1sakor.",
            "missing_configuration": "A komponens nincs konfigur\u00e1lva. K\u00e9rem, k\u00f6vesse a dokument\u00e1ci\u00f3t.",
            "no_url_available": "Nincs el\u00e9rhet\u0151 URL. A hib\u00e1val kapcsolatos inform\u00e1ci\u00f3k\u00e9rt n\u00e9zze meg a [s\u00fag\u00f3]({docs_url}) le\u00edr\u00e1s\u00e1t.",
            "oauth_error": "\u00c9rv\u00e9nytelen token adatok \u00e9rkeztek.",
            "oauth_failed": "Hiba a hozz\u00e1f\u00e9r\u00e9si token megszerz\u00e9se k\u00f6zben.",
            "oauth_timeout": "Id\u0151t\u00fall\u00e9p\u00e9s az OAuth-token felold\u00e1sakor.",
            "oauth_unauthorized": "OAuth enged\u00e9lyez\u00e9si hiba a hozz\u00e1f\u00e9r\u00e9si token megszerz\u00e9se sor\u00e1n.",
            "reauth_successful": "Az \u00fajrahiteles\u00edt\u00e9s sikeres volt.",
            "user_rejected_authorize": "A fi\u00f3k csatol\u00e1sa elutas\u00edtva: {error}"
        },
        "create_entry": {
            "default": "Sikeres hiteles\u00edt\u00e9s"
        },
        "error": {
            "cannot_connect": "Sikertelen csatlakoz\u00e1s",
            "invalid_auth": "\u00c9rv\u00e9nytelen hiteles\u00edt\u00e9s"
        },
        "step": {
            "pick_implementation": {
                "title": "V\u00e1lasszon egy hiteles\u00edt\u00e9si m\u00f3dszert"
            },
            "reauth_confirm": {
                "data": {
                    "password": "Jelsz\u00f3"
                },
                "description": "Az Aladdin Connectnek \u00fajra kell hiteles\u00edtenie fi\u00f3kj\u00e1t",
                "title": "A(z) {name} hiteles\u00edt\u00e9se lej\u00e1rt"
            },
            "user": {
                "data": {
                    "password": "Jelsz\u00f3",
                    "username": "Felhaszn\u00e1l\u00f3n\u00e9v"
                }
            }
        }
    },
    "entity": {
        "sensor": {
            "ble_strength": {
                "name": "BLE er\u0151ss\u00e9g"
            },
            "wifi_strength": {
                "name": "Wi-Fi RSSI"
            }
        }
    },
    "issues": {
        "integration_removed": {
            "description": "Az Aladdin Connect integr\u00e1ci\u00f3 el lett t\u00e1vol\u00edtva a Home Assistant rendszerb\u0151l.\n\nA probl\u00e9ma megold\u00e1s\u00e1hoz k\u00e9rj\u00fck, t\u00e1vol\u00edtsa el a (jelenleg nem m\u0171k\u00f6d\u0151) integr\u00e1ci\u00f3s bejegyz\u00e9seket a Home Assistant be\u00e1ll\u00edt\u00e1saib\u00f3l. [Kattintson ide, hogy megtekinthesse megl\u00e9v\u0151 Aladdin Connect integr\u00e1ci\u00f3s bejegyz\u00e9seit]({entries}).",
            "title": "Az Aladdin Connect integr\u00e1ci\u00f3 elt\u00e1vol\u00edt\u00e1sra ker\u00fclt"
        }
    }
}